tests/evaluate/test_default_evaluator.py

```diff
@@ -1986,12 +1986,52 @@ def language_model(inputs: list[str]) -> list[str]:
     return inputs
 
 
+def validate_question_answering_logged_data(logged_data, with_targets=True):
+    if with_targets:
+        columns = [
+            "question",
+            "answer",
+            "outputs",
+            "toxicity",
+            "flesch_kincaid_grade_level",
+            "ari_grade_level",
+            "perplexity",
+        ]
+    else:
+        columns = [
+            "question",
+            "outputs",
+            "toxicity",
+            "flesch_kincaid_grade_level",
+            "ari_grade_level",
+            "perplexity",
+        ]
+
+    assert logged_data.columns.tolist() == columns
+
+    assert logged_data["question"].tolist() == ["words random", "This is a sentence."]
+    assert logged_data["outputs"].tolist() == ["words random", "This is a sentence."]
+    assert logged_data["toxicity"][0] < 0.5
+    assert logged_data["toxicity"][1] < 0.5
+    assert logged_data["perplexity"][0] > logged_data["perplexity"][1]
+    for grade in logged_data["flesch_kincaid_grade_level"] + logged_data["ari_grade_level"]:
```

Comment:
Why do we take the sum of `flesch_kincaid_grade_level` and `ari_grade_level`?

Reply:
```
    print(logged_data["flesch_kincaid_grade_level"])
    print(logged_data["ari_grade_level"])
    print(logged_data["flesch_kincaid_grade_level"] + logged_data["ari_grade_level"])
```

```
0   -3.5
1   -3.5
Name: flesch_kincaid_grade_level, dtype: float64
0   -16.3
1   -16.3
Name: ari_grade_level, dtype: float64
0   -19.8
1   -19.8
dtype: float64
```

Reply:
Sorry about this 😅, I meant to append the two (like they are lists) but they are not lists. I'll fix this!